tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -509,6 +509,41 @@ def test_forward_gather():
     _test_gather((3,3,3), (1,1,2), [[[1,0]]], 2, 'int32')
     _test_gather((4,3,5,6), (1,4), [[2,1,0,0]], 0, 'float32')
 
+
+def _test_gather_v1(ip_shape, indice_shape, indice_value, dtype):
+    """ One iteration of a Gather"""
+    tf.reset_default_graph()
+    in_data = tf.placeholder(dtype, ip_shape, name="in_data")
+    indices = tf.placeholder("int32", indice_shape, name="indices")
+    tf.gather(in_data, indices)
+    np_data = np.random.uniform(size=ip_shape).astype(dtype)
+
+    def _fill_indices(indice_value):
+        indices = np.array(ip_shape, dtype=dtype)
+        if isinstance(indice_value, int):
+            indices = np.array([indice_value], dtype='int32')
+        else:
+            indices = np.asarray(indice_value, dtype='int32')
+        return indices
+    np_indices = _fill_indices(indice_value)
+
+    compare_tf_with_tvm([np_data, np_indices], ['in_data:0', 'indices:0'], 'Gather:0')
+
+def test_forward_gather_v1():
```

Comment:
```suggestion
from nose.tools import nottest

@nottest
def test_forward_gather_v1():
```

This will skip the unit test. Integ test is failing because it's running your test.

Reply:
how about adding if condition for the test? `if int(tf.__version__.split('.')[1]) <= 6:`

Reply:
sounds good. Checkout out an example: https://github.com/dmlc/tvm/blob/5f37cc1290355c840e36ba3a4a7ff158a97896c5/tests/python/contrib/test_nnpack.py#L20-L21